features/metal3/exec.config

```diff
@@ -24,11 +24,10 @@ rm -rf "$TEMP_DEB_DIR"
 apt-mark hold ${package_name}
 
 # crictl
```

Comment:
This installation part is done in feature `convergedcloud`. Expecting that `metal3` does not require it please remove the duplicated section from here.

Reply:
The `convergedcloud` feature is not used in `metal3` hence the installation is fine here. We will need an internal alignment in ccloud how to improve the current features.